Source/NSEPSImageRep.m

```diff
@@ -103,13 +123,16 @@ - (void) prepareGState
   // This is for subclasses only
 }
 
-// Drawing the Image 
+// Override to draw the specified page...
 - (BOOL) draw
 {
-  [self notImplemented: _cmd];
+  NSRect irect = NSMakeRect(0, 0, _size.width, _size.height);
+  NSGraphicsContext *ctxt = GSCurrentContext();
 
   [self prepareGState];
-
+  
+  [_pageRep _premultiply];
```

Comment:
Wouldn't it be simple to delegate the drawing to the _pageRep by [_pageRep draw] here?